ocaml/grammar.js

```diff
@@ -2025,10 +2025,10 @@ module.exports = grammar({
     _capitalized_identifier: $ => /[A-Z][a-zA-Z0-9_']*/,
 
     _label: $ => seq(choice('~', '?'), $._label_name),
-    directive: $ => seq('#', choice($._identifier, $._capitalized_identifier)),
-    type_variable: $ => seq("'", choice($._identifier, $._capitalized_identifier)),
-    tag: $ => seq('`', choice($._identifier, $._capitalized_identifier)),
-    attribute_id: $ => sep1('.', choice($._identifier, $._capitalized_identifier))
+    directive: $ => seq(/#/, choice($._identifier, $._capitalized_identifier)),
+    type_variable: $ => seq(/'/, choice($._identifier, $._capitalized_identifier)),
+    tag: $ => seq(/`/, choice($._identifier, $._capitalized_identifier)),
+    attribute_id: $ => sep1(/./, choice($._identifier, $._capitalized_identifier))
```

Comment:
this means any charachter. you probably need to quote it.

Reply:
Is the right solution to quote it `/"."/` as you suggest or to escape it `/\./`, as suggested in https://github.com/tree-sitter/tree-sitter-ocaml/issues/63#issuecomment-1350213196 ?

Reply:
escape, like what was proposed originally